DOCS-10591-jms-consume-operation-kt (#1398)
mulesoft/docs-connectors: jms/1.7/modules/ROOT/pages/jms-consume.adoc

> = To Consume JMS Messages

Reviewer: Type: CE
Scope: Review only the sections Wait for a Message and Poll for a Message, because other sections were already edited in a different PR (Publish Consume) and they are exactly the same.

</jms:consume>
----

> == Wait for a Message

Reviewer: HERE

> By default, the *Consume* operation has a maximum wait time configured of `10000` milliseconds, which produces a `JMS:TIMEOUT` error if no message is available in that period.

Reviewer: ```suggestion
By default, the *Consume* operation waits for a message for up to 10000 milliseconds, after which you receive the error `JMS:TIMEOUT`. 
```

> In order to configure a timeout that fits better your use case, configure the *Maximum wait* and *Maximum wait unit* fields.

Reviewer: GLOBAL "in order to" > "to"

Reviewer: ```suggestion
To better suit your use case, configure the *Maximum wait* and *Maximum wait unit* fields.
```

To wait indefinitely for a message to arrive, set the *Maximum wait* field to `-1`, which prevents a `JMS:TIMEOUT` error.

> == Poll for a Message

Reviewer: HERE

> When polling for a message with a certain frequency, you don't want the consumer to block waiting for the message, but instead just try to consume it if one is present.

Reviewer: "you" are not polling for a message, so this needs rewording to clarify, something like...:
```suggestion
To prevent message polling from blocking other actions as it waits for a message, set the *Maximum wait* field to `0`. This setting prevents both blocking and a timeout error. If polling produces no results, the *Consume* operation output is `null`.
```

> In this case, set the *Maximum wait* field to `0`, which indicates that no waiting occurs, and neither a `JMS:TIMEOUT` error is thrown if no message is available in the specified destination.

Reviewer: See above, or reword as you think best.
```suggestion
```

> The output of the *Consume* operation is `null` when no message is present for consumption and no error is thrown.

Reviewer: See above; ditto comment Line 76.
